react/features/base/util/loadScript.web.js

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
I propose to merge this file when we move conference initialization to react app